src/utils/juror-draft-utils.js

```diff
@@ -0,0 +1,24 @@
+import { addressesEqual } from '../lib/web3-utils'
+import { isvoteLeaked } from './crvoting-utils'
+
+export function getJurorDraft(round, jurorId) {
+  if (!round) return null
+
+  return round.jurors.find(jurorDraft =>
+    addressesEqual(jurorDraft.juror.id, jurorId)
+  )
+}
+
+export function jurorVoted(jurorDraft) {
```

Comment:
What would you think of using a verb here?

```suggestion
export function hasJurorVoted(jurorDraft) {
```